week-view-lib/src/main/java/com/alamkanak/weekview/WeekView.java

```diff
@@ -988,8 +1000,16 @@ private void drawEvents(Calendar date, float startFromPixel, Canvas canvas) {
                         mEventRects.get(i).rectF = new RectF(left, top, right, bottom);
                         mEventBackgroundPaint.setColor(mEventRects.get(i).event.getColor() == 0 ? mDefaultEventColor : mEventRects.get(i).event.getColor());
                         mEventBackgroundPaint.setShader(mEventRects.get(i).event.getShader());
-                        canvas.drawRoundRect(mEventRects.get(i).rectF, mEventCornerRadius, mEventCornerRadius, mEventBackgroundPaint);
+                        if (isSameDayAndTimeBetween(mEventRects.get(i).event.getStartTime(), mEventRects.get(i).event.getEndTime())) {
+                            canvas.drawRoundRect(mEventRects.get(i).rectF, mEventCornerRadius, mEventCornerRadius, mEventBackgroundPaint);
+                            canvas.drawRoundRect(mEventRects.get(i).rectF, mEventCornerRadius, mEventCornerRadius, mNowBackgroundPaint);
```

Comment:
Čia nėra geras sprendimas, geriau paimk event'o spalvą ir pakoreguok (pašviesink/patamsink/patint'ink), po to pakeisk su `mEventBackgroundPaint.setColor`